packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -481,9 +513,16 @@ contract Colosseum_Initializer is L2OutputOracle_Initializer {
 }
 
 contract FFIInterface is Test {
-    function getProveWithdrawalTransactionInputs(
-        Types.WithdrawalTransaction memory _tx
-    ) external returns (bytes32, bytes32, bytes32, bytes32, bytes[] memory) {
+    function getProveWithdrawalTransactionInputs(Types.WithdrawalTransaction memory _tx)
+        external
+        returns (
+            bytes32,
+            bytes32,
+            bytes32,
+            bytes32,
+            bytes[] memory
+        )
+    {
```

Comment:
Could you restore to the original code? The original code is the right code which conforms to formatting rule.

Reply:
@0xHansLee I'm not sure why my pretier is changing the format of this code, could you take a look at this offline?

Reply:
Oh! If I ran the command below offline, I found you're right. Please ignore my comment.

```shell
yarn prettier --write contracts/test/CommonTest.t.sol
```